src/Microsoft.AspNetCore.Routing/DispatcherMiddleware.cs

```diff
@@ -101,14 +105,24 @@ private Task<Matcher> InitializeAsync()
         private static class Log
         {
             private static readonly Action<ILogger, string, Exception> _matchSuccess = LoggerMessage.Define<string>(
-                LogLevel.Information,
-                new EventId(0, "MatchSuccess"),
+                LogLevel.Debug,
+                new EventId(1, "MatchSuccess"),
```

Comment:
What number should event ID start at? On SignalR we always started at 1, and I like the idea of leaving zero alone.